src/extensions/registries/page-registry.ts

```diff
@@ -1,31 +1,33 @@
 // Extensions-api -> Custom page registration
 
-import type React from "react";
-import type { RouteProps } from "react-router";
-import type { IconProps } from "../../renderer/components/icon";
-import type { IClassName } from "../../renderer/utils";
-import type { TabRoute } from "../../renderer/components/layout/tab-layout";
-import { BaseRegistry } from "./base-registry";
+import React from "react";
+import { BaseRegistry, BaseRegistryItem } from "./base-registry";
 
-export interface PageRegistration extends RouteProps {
-  className?: IClassName;
-  url?: string; // initial url to be used for building menus and tabs, otherwise "path" applied by default
-  title?: React.ReactNode; // used in sidebar's & tabs-layout if provided
-  hideInMenu?: boolean; // hide element within app's navigation menu
-  subPages?: (PageRegistration & TabRoute)[];
+export interface PageRegistration extends BaseRegistryItem {
+  routePath?: string; // additional (suffix) route path to base extension's route: "/extension/:name"
+  exact?: boolean; // route matching flag, see: https://reactrouter.com/web/api/NavLink/exact-bool
```

Comment:
This is a bit mystery flag ... when does it needs to be set?

Reply:
I was hoping provided url from comment will explain this.. but.. it's about strict matching route:
e.g. if you have route like `/page` with `exact: true` it will match only this, but not `/page/sub-page`